lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -107,7 +107,8 @@ function LoggedInUser({ userInformation, onChangeCallback, submitForm, showWarni
           <div className="col-sm-12">
             <Button
               className={['btn', 'btn-primary', 'btn-submit']}
-              type="submit"
+              type="button"
+              onClick={submitForm}
```

Comment:
Rest is fine. I am just curious if change of button type is necessary here. I think If we keep the button type to submit along with your other code changes, it should work too. **submitForm** function would get the target event and we can move forward by enabling/disabling submit button by adding or removing attributes as we are currently doing. We would be doing minimal change to make it work.

Reply:
@adeelkhan If we keep the button type submit and not bind the `onClick` to `submitForm`, we get the `form` object in event target instead of the button. That's the reason I had to change it.

Reply:
Got it!. thanks for explaining. Just a comment. Going through the code again seems to suggest, that there were no form submission via html prior to your change ie submit call was blocked and form info is fetched from formData and send via ajax. It seems more appropriate to rename submitForm function to something like **sendRequestToZendesk**. or **sendFormData**

Reply:
Changed it to `handleClick` which I believe makes more sense.